### PR TITLE
Temporary field guide page

### PR DESCRIPTION
**Pull request by jforgue**

### CODE REVIEW

**Comment by dfaulken** [comment]
Screenshot?

**Comment by jforgue** [comment]
screen shot 2018-06-14 at 5 44 37 pm
screen shot 2018-06-14 at 5 44 55 pm
screen shot 2018-06-14 at 5 45 28 pm

File: app/views/bus_stops/field_guide.haml [hunk 1]
```diff
@@ -0,0 +1,447 @@
+.page-title Field Guide
+
+%strong Materials needed
+– you will need a tape measure, measuring wheel, and camera.
+%br
```
Comment (dfaulken) [inline app/views/bus_stops/field_guide.haml]: I would favor styling to br tags.

**Comment by jforgue** [comment]
I should mention I have some code for tooltips to better solve #52 but I'll hold off on completing that until #55 is taken care of.

**Comment by mboneil10** [comment]
Was the field guide asked to be at the end of the entry page? I think it would make most sense to have it at the top, so if a user really has no idea what's happening they don't have to scroll to the bottom of a page.

**Comment by mboneil10** [comment]
Answered by @sherson

**Comment by mboneil10** [comment]
Meh my changes don't disturb anything other than `u`s. And `u`s are only used on this page.

**Pull request merged**